src/components/Card/index.tsx

```diff
@@ -0,0 +1,108 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */ //remover essa linha depois
+
+import React, { useState } from 'react';
```

Comment:
na versão atual do React neste projeto, não é preciso importar o React - [confira](https://reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html)

```suggestion
import { useState } from 'react';
```